packages/micropip/test_micropip.py

```diff
@@ -281,6 +282,23 @@ def test_add_requirement_marker(mock_importlib):
     assert len(transaction.wheels) == 1
 
 
+def test_add_install_with_extra(mock_importlib):
+    pytest.importorskip("packaging")
+    from micropip._micropip import Transaction
+
+    transaction = create_transaction(Transaction)
+
+    asyncio.get_event_loop().run_until_complete(
+        transaction.gather_requirements(
+            [
+                "hypothesis[cli]",
```

Comment:
Ideally, i would like to update the `pyodide-micropip-test` package here updated with some extras.